zboss/include/zboss_api_zgp.h

```diff
@@ -1502,7 +1502,7 @@ void zb_zgps_set_communication_mode(zgp_communication_mode_t mode);
  * Basic specification v1.1.1, tables 51, 52). If there is no default
  * translation found, then the received command is dropped.
  *
- * If this function is implemented by the the application, then application is
+ * If this function is implemented by the application, then application is
```

Comment:
This change should be done in the upstream. Even if we fix it now, it will be overwritten by the next ZBOSS update.